homeassistant/components/cookidoo/strings.json

```diff
@@ -2,25 +2,25 @@
   "config": {
     "step": {
       "user": {
-        "title": "Login to Cookidoo",
+        "title": "Login to {cookidoo}",
         "data": {
           "email": "[%key:common::config_flow::data::email%]",
           "password": "[%key:common::config_flow::data::password%]",
           "country": "Country"
         },
         "data_description": {
-          "email": "Email used access your Cookidoo account.",
-          "password": "Password used access your Cookidoo account.",
-          "country": "Pick your language for the Cookidoo content."
+          "email": "Email used to access your {cookidoo} account.",
+          "password": "Password used to access your {cookidoo} account.",
+          "country": "Pick your language for the {cookidoo} content."
         }
       },
       "language": {
-        "title": "Login to Cookidoo",
+        "title": "Login to {cookidoo}",
```

Comment:
Should this title be adjusted?

Reply:
@epenet Maybe, to what? "Pick your language for {cookidoo}"?

Reply:
Otherwise it is the title of the dialog as is, and in data_description is enough information for the user what is expected of him, no?